src/python/grpcio/grpc/_server.py

```diff
@@ -793,30 +911,33 @@ def __init__(self, completion_queue, server, generic_handlers,
         self.server_deallocated = False
 
 
-def _add_generic_handlers(state, generic_handlers):
+def _add_generic_handlers(
+        state: _ServerState,
+        generic_handlers: Iterable[grpc.GenericRpcHandler]) -> None:
```

Comment:
`Sequence`? The callers all offer `Sequence` and that would give us more flexibility when changing this function in the future.

Reply:
[add_generic_rpc_handlers](https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/__init__.py#L1407) says it's expecting an iterable in public documentation, since not all `Iterable` are `Sequence`, change it to sequence might break some integration?

Reply:
That's true. We _could_ convert to a sequence by doing `list(user_input)` somewhere near the top, but this is fine for now.